src/amo/components/GuidesAddonCard/styles.scss

```diff
@@ -49,6 +49,12 @@
   @include respond-to(medium) {
     flex-wrap: nowrap;
   }
+
+  .GetFirefoxButton {
+    font-size: $font-size-default;
+    text-align: center;
+    white-space: normal;
+  }
 }
```

Comment:
This is a general comment, not something to change here: it'd be nice to only have one CSS block per main CSS class name and not some in `.GuidesAddonCard` and some below.